geometry_msgs/PoseStamped gps
float32[] nodePointDistances
---
PolynomialCoeffs[] coefficients
float32[] kappa